tools/ubuntu-setup/misc.sh

```diff
@@ -22,7 +22,7 @@ set -x
 export DEBIAN_FRONTEND=noninteractive
 
 sudo apt-get update -y
-sudo apt-get install -y ntp git zip tzdata lsb-release
+sudo apt-get install -y ntp git zip tzdata lsb-release npm
```

Comment:
fwiw, I've had to also install `unzip` for the digital ocean droplets.

Reply:
ok, let's add `unzip` to the apt-get as well.

Reply:
done.